test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py

```diff
@@ -104,6 +106,10 @@ def test_stop_cache_with_mounted_partition():
         core = cache.add_core(core_part)
         core.mount(mount_point)
 
+    with TestRun.step("Ensure /etc/mtab exists."):
+        if not fs_utils.check_if_symlink_exists("/etc/mtab"):
+            Symlink.create_symlink("/proc/self/mounts", "/etc/mtab")
```

Comment:
Are you sure every distro links mtab to /proc/self/mounts ? :)

Reply:
I'm not - I'll change it so this checks if any type of /etc/mtab exists. If it does we'll leave it alone, if not then creating such a symlink shouldn't break anything ;)